### ISSUE TITLE
Normal swipes not working anymore.

### ISSUE THREAD

**Issue by matt1432** [horriblename/hyprgrass#76]
Since the edge swiping feature was added, I can't do normal workspace swipes. The edge swiping also seems a bit hard to get to work.

**Comment by horriblename** [comment]
does workspace swipe completely stop working or something else? 

are you sure the edge swiping pr caused it? And not the long press pr #70? 

---

I just realized the base threshold for edge swipe is ridiculously high, fixed in #80

**Comment by matt1432** [comment]
It definitely comes from this commit: https://github.com/horriblename/hyprgrass/commit/60bf0c0eef0e506f0291f9b5f7b5d33c56a5efb1

**Comment by matt1432** [comment]
Here's a video to show the before and after

https://github.com/horriblename/hyprgrass/assets/98562002/fe0af174-409d-4008-9777-ba1a61d57589

**Comment by matt1432** [comment]
> I just realized the base threshold for edge swipe is ridiculously high, fixed in https://github.com/horriblename/hyprgrass/pull/80

I also tested this and it is perfect now btw!

**Comment by horriblename** [comment]
please send config, if possible reduced to hyprgrass options + the builtin `gestures` options. and also re-send the video, it seems to be corrupted.

**Comment by matt1432** [comment]
```
plugin {
  touch_gestures {
    # The default sensitivity is probably too low on tablet screens,
    # I recommend turning it up to 4.0
    sensitivity = 4.0

    # must be >= 3
    workspace_swipe_fingers = 3

    experimental {
      # send proper cancel events to windows instead of hacky touch_up events,
      # NOT recommended as it crashed a few times, once it's stabilized I'll make it the default
      send_cancel = 0
    }
  }
}

gestures {
    workspace_swipe = yes
    workspace_swipe_fingers = 3
    workspace_swipe_cancel_ratio = 0.15
}
```
https://github.com/horriblename/hyprgrass/assets/98562002/00ae34f5-0c88-4984-a62d-c70198649cad

**Comment by horriblename** [comment]
should be fixed now (#83)

test pls

**Comment by matt1432** [comment]
Tried the plugin again after some time and this has been fixed sinced then
